integration_test/ops_agent_test.go

```diff
@@ -1761,12 +1766,40 @@ func testDefaultMetrics(ctx context.Context, t *testing.T, logger *logging.Direc
 		}()
 	}
 	metricsWaitGroup.Wait()
+
+	logger.ToMainLog().Printf("Attempting to read features.yaml\n")
+	featureBytes, err := os.ReadFile(path.Join("agent_metrics", "features.yaml"))
+	if err != nil {
+		t.Fatalf("Could not find features.yaml\n")
+		return
+	}
+
+	var fc feature_tracking.FeatureTrackingContainer
+	logger.ToMainLog().Printf("Read features.yaml successful\n")
```

Comment:
You generally don't need `Printf` or `Fatalf` if you're not formatting any text. Just use `Println` and `Fatal` without the `\n`.